website/docs/docs/dbt-versions/core-upgrade/05-upgrading-to-v1.10.md

```diff
@@ -51,35 +51,22 @@ You can read more about each of these behavior changes in the following links:
 
 ### Deprecation warnings
 
-New deprecation warnings in v1.10.
-
-#### Duplicate keys in profiles.yml
+Starting in v1.10, you will receive deprecation warnings for dbt code that will become invalid in the future, including: 
 
+- Custom inputs (for example, unrecognized resource properties, configurations, and top-level keys)
+- Duplicate YAML keys in the same file
+- Unexpected jinja blocks (for example, `{% endmacro %}` tags without a corresponding `{% macro %}` tag)
+- And more
 
-In v1.10, if two identical keys exist in the `profiles.yml`, you will get a warning, and in a future version, dbt will stop supporting duplicate keys with silent overwrite. 
-Previously, if identical keys existed in the [`profiles.yml` file](/docs/core/connect-data-platform/profiles.yml), dbt would use the last configuration listed in the file. 
+dbt will start raising these warnings in version 1.10, but making these changes will not be a prerequisite for using it. We at dbt Labs understand that it will take existing users time to migrate their projects, and it is not our goal to disrupt anyone with this update. The goal is to enable you and your dbt users to work with more safety, feedback, and confidence going forward.
```

Comment:
```suggestion
dbt will start raising these warnings in version `1.10`, but making these changes will not be a prerequisite for using it. We at dbt Labs understand that it will take existing users time to migrate their projects, and it is not our goal to disrupt anyone with this update. The goal is to enable you to work with more safety, feedback, and confidence going forward.
```